.github/workflows/build.yml

```diff
@@ -0,0 +1,92 @@
+---
+
+name: Build wheels
+on: push
+jobs:
+  manylinux_wheels:
+    name: Python ${{ matrix.python_tag }} ${{ matrix.platform }} wheels
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform:
+          - manylinux1_x86_64
+          - manylinux2014_x86_64
```

Comment:
Why skip 2010? Also, if auditwheel run under 2014 decides that the wheel is worthy of manylinux1, it'll generate those wheels too, regardless...

Reply:
The only reason for skipping 2020 was that this PR started off with just manylinux2014 wheels (to the best of my memory, I don't remember that manylinux1 wheels where built automatically..), and then someone responded making the valid point that the platform pip version on some OSs doesn't support 2010 or 2014, so I added the manylinux1 wheels in explicitly seeing if they could be built as well.

If auditwheel will build manylinux1 (and manylinux2010?) automatically, that's good then and this can be simplified.

Reply:
Strictly speaking, auditwheel doesn't "build" wheels, it inspects the shared objects in the prebuilt ones with `readelf`, then repacks those wheels with proper names and metadata, and if necessary, copies in the dynamically linked libraries so that they are vendored + rewrites so files to link against those vendored libs.
If you run it in 2014 mode, it'll do the above and check what "allowed" libs it's linked against. If the links are against things that are also allowed for 2010, it'll create those too, same for "1". Here's a lot example: https://github.com/ansible/pylibssh/runs/1504752912#step:10:1039 (the actual build container is manylinux1).
> INFO:auditwheel.main_repair:Wheel is eligible for a higher priority tag. You requested manylinux2010_x86_64 but I have found this wheel is eligible for manylinux1_x86_64.
> [...]
> INFO:auditwheel.main_repair:Wheel is eligible for a higher priority tag. You requested manylinux2014_x86_64 but I have found this wheel is eligible for manylinux1_x86_64.